src/exercises/debug-and-disassemble/answers.md

```diff
@@ -122,3 +122,131 @@ The capability points at:
           0x40193000         0x4028c000    0xf9000    0x85000 r-x C--- /lib/libc.so.7
 ```
 12. Left as an exercise to the reader.
+
+# Answers - Disassemble and debug aarch64 and Morello programs
+
+2. `bl`. The target address is a pc-relative address in the `.plt`
+   section addressed by a sequence like:
+```
+   210abc: 31 00 00 94  bl	0x210b80 <printf@plt>
+; 	printf("size of address: %zu\n", sizeof(size_t));
+
+```
+3. `bl`. The target capability is loaded from the `.captable` section
+   by a sequence like:
+```
```

Comment:
Needs filling out, and there's a good chance the test isn't actually loading from the GOT and is instead deriving the capability for the string constant inline from PCC